proguard.pro

```diff
@@ -44,6 +44,6 @@
 -keep,allowoptimization class io.github.prospector.modmenu.** { *; } # ugly classloader hack
 
 # Fabric entrypoints
--keep,allowoptimization class dev.nolij.zume.FabricZumeBootstrapper
--keep,allowoptimization class dev.nolij.zume.modern.integration.ZumeModMenuIntegration
--keep,allowoptimization class dev.nolij.zume.primitive.event.KeyBindingRegistrar { public *; }
+-keep,allowoptimization,allowobfuscation class dev.nolij.zume.FabricZumeBootstrapper
+-keep,allowoptimization,allowobfuscation class dev.nolij.zume.modern.integration.ZumeModMenuIntegration
+-keep,allowoptimization,allowobfuscation class dev.nolij.zume.primitive.event.KeyBindingRegistrar { public *; }
```

Comment:
Please ensure class members being renamed doesn't cause bugs.

Reply:
seems to work for me (I only tested in-game on modern, but the game loaded on primitve too)

Reply:
Please also check that the keybinds register in Primitive, and that the Mod Menu integration actually works in Modern (ie that the config button is registered and opens config file on click).

Reply:
Mod(ern) Menu integration works fine. I can't test primitive because macOS just won't launch fabric, but given that MM works I would assume that it does too.